docs/DOCKER.md

```diff
@@ -7,19 +7,26 @@ check out the documentation at [docker/README.md](../docker/README.md).
 
 ## Production
 
-To use Docker in a production environment, you can run `lnd` by first creating
-a Docker container, adding the appropriate command-line options as parameters.
+To use Docker in a production environment, you can run `lnd` by creating a
+Docker container, adding the appropriate command-line options as parameters.
+
+You first need to build the lnd docker image:
 
 ```
-$ docker create --name=lnd lightninglabs/lnd [command-line options]
+$ git checkout <tag-or-branch>
```

Comment:
I created a PR to allow the docker image to be built with a tag or commit a while back: https://github.com/lightningnetwork/lnd/pull/2520/files.  Is there a reason you prefer a different PR and approach?

Reply:
Yes, there is a reason.

With your approach, you need two dockerfiles, one to build from local state, and another one to build for a tag / branch.

As everyone know, programmers are lazy and don't want to maintain two dockerfiles :)

Reply:
yes, but what if someone wishes to build from github without checking out this repo?

Reply:
Let's continue on [this other comment thread](https://github.com/lightningnetwork/lnd/pull/2649#discussion_r257935874) where we discuss the same points.